homeassistant/components/amcrest/camera.py

```diff
@@ -188,6 +189,7 @@ async def _async_get_image(self) -> None:
                     self._api.snapshot,
                     timeout=(COMM_TIMEOUT, SNAPSHOT_TIMEOUT),
                     stream=False,
+                    channel=self._channel + 1,
```

Comment:
I'm not familiar with this device, why are some of the references `_channel + 1` while others are  `_channel`?

Maybe consider creating a second variable set to `_channel + 1` so you can give it a descriptive name